commands/daily.py

```diff
@@ -30,6 +30,13 @@ async def daily(self, ctx):
             multiplier = 1
 
         if last_advent < current_time:
+            
+            def update_advent(user_id: str):

```

Comment:
Why is this method under the if statement? Refer to coinflip command for how this should be implemented.

Reply:
user_id does not need to be passed in, just directly use ctx.author.id in the method